onnxscript/tests/functions/onnxfns2_test.py

```diff
@@ -116,7 +116,7 @@ def test_onnxfns_hardmax(self):
     #         skip_test_names=[])
 
     @unittest.skipIf(
-        Version(onnxruntime.__version__) == Version("1.14"),
+        Version(onnxruntime.__version__) == Version("1.14.1"),
```

Comment:
Check the major, minor version to make it more robust maybe?

Reply:
I used `onnxruntime.__version__[:4] == "1.14",` instead. PTAL. Feel free to merge this PR if you are good with the updates.